backend/de.metas.manufacturing/src/main/java/de/metas/manufacturing/generatedcomponents/MacAddressGenerator.java

```diff
@@ -111,7 +113,8 @@ public ImmutableAttributeSet generate(@NonNull final ComponentGeneratorContext c
 		{
 			return ImmutableAttributeSet.EMPTY;
 		}
-		if (countRemainingToGenerate > attributesAvailableToGenerate.size())
+		final int availableSize = attributesAvailableToGenerate.size();
```

Comment:
hi, it generally looks OK to me, i'm just a bit concerned about the understandability..so wdyt about
- does it make sense to create the `-`  Mac-Addresses only for mandatory attributes?
- add a javadoc to this method that describes that missing Mac-Addresses with be `-`ed

Reply:
* The value "-" is set for the MAC Attributes even if they are not mandatory for manufacturing.
* Column "Description"added to PP_ComponentGenerator and displayed in the window